### PR TITLE
16: Routing I - Login (Part 1)

### PR DESCRIPTION
**Pull request by gabrielizalo**
# Summary

* First task of Routing: Login and using Router to the Game

# Trello Card

https://trello.com/c/uRUucKHu/9-routing-i-login

# Screenshot

login-1

login2

### CODE REVIEW

File: react/react-training/src/app/components/AppRoutes/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import Game from '@screens/Game';
+import Login from '@screens/Login';
+import { ROUTES } from '@constants/routes.js';
+
+export default class AppRoutes extends Component {
+  render() {
+    return (
+      <Router>
+        <div>
```
Comment (henryzarza) [inline react/react-training/src/app/components/AppRoutes/index.js]: You don't need this `div`

File: react/react-training/src/app/screens/Login/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+import styles from './styles.scss';
+
+class Login extends Component {
```
Comment (henryzarza) [inline react/react-training/src/app/screens/Login/index.js]: You can change this for `functional` component

**Review by henryzarza — CHANGES_REQUESTED** [review]

File: react/react-training/src/app/components/AppRoutes/index.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import Game from '@screens/Game';
+import Login from '@screens/Login';
+import { ROUTES } from '@constants/routes.js';
+
+export default class AppRoutes extends Component {
+  render() {
+    return (
+      <Router>
+        <Fragment>
```
Comment (wfolini) [inline react/react-training/src/app/components/AppRoutes/index.js]: Use `Switch` https://reacttraining.com/react-router/core/api/Switch

**Review by wfolini — CHANGES_REQUESTED** [review]
Great, Small change!

**Review by wfolini — APPROVED** [review]

**Review by henryzarza — APPROVED** [review]

**Pull request merged**